apis/types/types.go

```diff
@@ -148,3 +148,8 @@ const (
 	// HelmRepository is the config type for Helm chart repository
 	HelmRepository = "config-helm-repository"
 )
+
+const (
+	// TerrfaormComponentPrefix is the prefix of component type of terraform-xxx
+	TerrfaormComponentPrefix = "terraform-"
```

Comment:
```suggestion
	TerrfaormComponentPrefix = "tf-"
```

K8s name has a limit of 64 char, we should make the common prefix short

Reply:
They are names used as labels to mark the application which works as the controller to create a Terraform provider config.